python/interpret_community/explanation/explanation.py

```diff
@@ -724,7 +724,11 @@ def data(self, key=None):
                 for local_mli_data in mli_data:
                     if local_mli_data[InterpretData.EXPLANATION_TYPE] == InterpretData.LOCAL_FEATURE_IMPORTANCE:
                         local_mli_data[InterpretData.VALUE][InterpretData.INTERCEPT] = self.expected_values
+                class_dimension = InterpretData.SINGLE
+                if PerClassMixin._does_quack(self):
+                    class_dimension = InterpretData.MULTICLASS
```

Comment:
I think we should leave this as optional and not even write it in these cases. Since these don't override the model dimension, they are redundant. Reserving the ability to add this later

Reply:
Yeah this looks completely redundant right now if the only thing we care about is whether it has PerClassMixin

Reply:
done, only set class dimension if single dimension as discussed